YoCode/DuplicationCheck.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace YoCode
+{
+    class DuplicationCheck
+    {       
+        string CMDtoolsDir = @"C:\Users\ukmzil\source\repos\Tools\CMD";
```

Comment:
This is the thing we're going to get from a configuration file, right? It's just here temporarily?

Reply:
yes